docs/workflow/ci/failure-analysis.md

```diff
@@ -0,0 +1,108 @@
+    - The field `ExcludeConsoleLog` should be `true` as much as possible. Setting it to false will mean all failures appear in the same log and since you didn't exclude the overall logs, all failures in the collection will get attributed to this one's issue bucket and errors will not be investigated properly. Due to limitations in Known Issues around rate limiting and xUnit resiliency, setting `ExcludeConsoleLog=false` is necessary in two scenarios:
+      + Nested tests as reported to Azure DevOps. Essentially this means theory failures, which look like this when reported in Azure DevOps: ![xUnit theory seen in azure devops](theory-azdo.png).
+        Adding support for this requires too many API calls, so using the console log here is necessary.
+      + Native crashes in libraries also require using the console log. This is needed as the crash corrupts the test results to be reported to Azure DevOps, so only the console logs are left.
+    - Optionally you can add specifics as needed like leg, configuration parameters, available dump links.
+
+Once the issue is open, feel free to rerun the `Build Analysis` check and the issue should be recognized as know if all was filed correctly. there are some known limitations as previously described. It also has no support at looking at fields outside the error message, the stacktrace, and the console log in the helix queue, so this approach doesn't yet cover devices too well yet.
+
+After you do this, if the failure is occuring frequently as per the data captured in the recently opened issue, please disable the failing test(s) with the corresponding issue link tracking the disable in a follow-up Pull Request
+
+* Update the tracking issue with the label `disabled-test`.
+* For libraries tests add a [`[ActiveIssue(link)]`](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/ActiveIssueAttribute.cs) attribute on the test method. You can narrow the disabling down to runtime variant, flavor, and platform. For an example see [File_AppendAllLinesAsync_Encoded](https://github.com/dotnet/runtime/blob/cf49643711ad8aa4685a8054286c1348cef6e1d8/src/libraries/System.IO.FileSystem/tests/File/AppendAsync.cs#L74)
+* For runtime tests found under `src/tests`, please edit [`issues.targets`](https://github.com/dotnet/runtime/blob/main/src/tests/issues.targets). There are several groups for different types of disable (mono vs. coreclr, different platforms, different scenarios). Add the folder containing the test and issue mimicking any of the samples in the file.
+
```

Comment:
Should we specify here whether they should disable the test in their same PR, or a separate one?

Reply:
The text specifies: 

> please disable the failing test(s) with the corresponding issue link tracking the disable *in a follow-up Pull Request*

It's probably better to be able to revert the one commit in the issue solution PR.

Reply:
Oops my bad, I didn't realize at the time it was specified above. Thanks for the clarification Juan